search/includes/classes/class-versioning.php

```diff
@@ -9,6 +9,10 @@
 
 class Versioning {
 	const INDEX_VERSIONS_OPTION = 'vip_search_index_versions';
+	/**
+	 * The maximum number of index versions that can exist for any indexable.
+	 */
+	const MAX_VERSIONS = 2;
```

Comment:
Minor note, but I would call this something more obvious like `MAX_NUMBER_OF_VERSIONS`. At a glance, `MAX_VERSIONS` could mean a number of different things and the more obvious naming avoids the ambiguity.

Reply:
Resolved via GitHub UI